index.js

```diff
@@ -44,13 +44,15 @@ const linux = name => {
 	};
 };
 
-module.exports = name => {
+module.exports = (name, suffix) => {
```

Comment:
Can you make suffix and option in an options object instead. To make it easier to add other options in the future.